Tests/SwiftSyntaxBuilderTest/SourceFileTests.swift

```diff
@@ -0,0 +1,29 @@
+import XCTest
+import SwiftSyntax
+import SwiftSyntaxBuilder
+
+final class SourceFileTests: XCTestCase {
+  func testSourceFile() {
+    let source = SourceFile(eofToken: .eof, statementsBuilder: {
+      ImportDecl(path: "Foundation")
+      ImportDecl(path: "UIKit")
+      ClassDecl(classOrActorKeyword: .class, identifier: "SomeViewController", members: MemberDeclBlock(membersBuilder: {
+        VariableDecl(.let, name: "tableView", type: "UITableView")
+      }))
```

Comment:
I think it would be great if this would look like
```suggestion
      ClassDecl(classOrActorKeyword: .class, identifier: "SomeViewController") {
        VariableDecl(.let, name: "tableView", type: "UITableView")
      }
```

What do you think of adding a convenience initializer to `ClassDecl` (probably also for `StructDecl`, `ProtocolDecl` etc.) that takes a `MemberDeclListBuilder`? I think we’d need to hand-craft that convenience initializer because I can’t think of a way to teach our code-generation to generate result builder parameters based on `ExpressibleAs` conformances.

Reply:
We need the last parameter `membersBuilder` otherwise we get `Backward matching of the unlabeled trailing closure is deprecated; label the argument with 'membersBuilder' to suppress this warning`

Reply:
I have added multiple extensions where there was no builder.